docs/metrics/README.md

```diff
@@ -417,6 +417,73 @@ SDK to reclaim unused metric points.
   Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
   become stable, this feature will be turned on by default.
 
+#### Cardinality overflow example
+
+In the following example we set the maximum number of `MetricPoint`s allowed to
+be `3` for two metrics. This means that for these metrics, the SDK will only
+export measurements for up to `3` distinct key/value combinations.
+
+> [!NOTE]
+> One `MetricPoint` is reserved on every `Metric` for the special case where
+  there are no key/value pairs associated with a measurement. When choosing a
+  cardinality limit users should account for this special case.
+
+```csharp
+Counter<long> MyFruitCounter = MyMeter.CreateCounter<long>("MyFruitCounter");
+Counter<long> AnotherFruitCounter = MyMeter.CreateCounter<long>("AnotherFruitCounter");
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter("*")
+    .AddConsoleExporter()
+    .AddView(i =>
+    {
+        if (i.Name == "MyFruitCounter" || i.Name == "AnotherFruitCounter")
+        {
+            // Note: Set the cardinality limit for 'MyFruitCounter' &
+            // 'AnotherFruitCounter' to 3
+            return new MetricStreamConfiguration { CardinalityLimit = 3 };
+        }
+
+        return null;
+    })
+    .Build();
+
+// There are four distinct key/value combinations emitted for 'MyFruitCounter':
+//   1. No key/value pair
+//   2. (name:apple, color:red)
+//   3. (name:lemon, color:yellow)
+//   4. (name:apple, color:green)
+// Since the cardinality limit is 3, the SDK will only export measurements for the following three combinations:
```

Comment:
Need to refine the wording, measurements are not exported at all. It is metric point which is being emitted https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exportbatch.

Reply:
Tweaked